canonical/charmhub.io
# Move publisher settings into React app

## Done
Moved the publisher settings page into the publisher React application

## How to QA
- Go to https://charmhub-io-1808.demos.haus/<CHARM_NAME>/settings
- Check that it has the same content as the live version

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-10530
